extensions/adobe/experience/profile/profile-all.schema.json

```diff
@@ -108,6 +108,86 @@
               }
             }
           }
+        },
```

Comment:
@patnaiku , I know you followed the previous way to adding the consent map to this field group. We can just add the field group ID ("$ref": "https://ns.adobe.com/xdm/mixins/profile-consentResults-v2)  to the allOf section of this field group. Remove the inline property. That way, it will ensure any change in the field group for consent will be propagated automatically to this.